repositories/memory/topics/get_topic_test.go

```diff
@@ -0,0 +1,98 @@
+package topics
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
```

Comment:
This import should be with the other third-party libs.

Reply:
Ack